samples/net/zperf/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+source "Kconfig.zephyr"
+
+config NET_CODE_RELOCATE
```

Comment:
I would prefer that the networking sample own Kconfig symbols are named so that there is no possibility to mix them with the core stack symbols. This is followed in other samples too.
So prefer:
```
config NET_SAMPLE_CODE_RELOCATE
```
and
```
config NET_SAMPLE_CODE_RAM_NAME
```

Reply:
Thanks for the suggestion- updated to use these names